db/seeds.rb

```diff
@@ -5,3 +5,14 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'nokogiri'
+require 'open-uri'
+
+page = Nokogiri::HTML(open("http://www.homeenglish.ru/250Popular1.htm"))
+a=14
+100.times do 
+  orig=page.css('td')[a].text 
+  transl=page.css('td')[a+1].text
```

Comment:
Surrounding space missing for operator `=`.<br>Surrounding space missing for operator `+`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.